src/api/link_post.js

```diff
@@ -44,16 +84,58 @@ class LinkPostHandler {
       return
     }
 
-    // Get address from sigsignature + msg
-    const address = await this.sigMgr.verify(msg, sig)
-    const consent = {
-      sig: sig,
-      msg: msg
+    return {msg, sig, did}
+  }
+
+  async v1Handler(body, cb) {
+    // Check if data is present
+    if (!body.signature) {
+      cb({ code: 403, message: 'no signature' })
+      return
+    }
+    if (!body.message) {
+      cb({ code: 403, message: 'no message' })
+      return
     }
 
-    await this.linkMgr.store(address, did, JSON.stringify(consent))
+    const sig = body.signature
+    const msg = body.message
+    const entry = body.entry
```

Comment:
Why is the entry passed along here? Seems like verifying the signature of the ethereum key should be enough?